app/screens/edit_post/edit_post.tsx

```diff
@@ -50,6 +56,9 @@ const styles = StyleSheet.create({
 
 const RIGHT_BUTTON = buildNavigationButton('edit-post', 'edit_post.save.button');
 
+// Exclude bottom edge from SafeAreaView to prevent gap between attachments and keyboard.
+const safeAreaEdges: Edge[] = ['top', 'left', 'right'];
```

Comment:
It is weird the reason is this. The safe area edges should add some space only to avoid things in the screen (like the camera).

Are you sure there is no extra margin style messing with things?

Reply:
Yes, I have checked, and there are no margin styles.